docs/hello-neighbour.yaml

```diff
@@ -184,212 +224,130 @@ paths:
-        name: "questionUuid"
-        type: "string"
-        format: "uuid"
+      - name: "churchUuid"
+        in: "path"
         required: true
-      responses:
-        204:
-          description: "Question deleted"
-        404:
-          description: "Question not found"
-          schema:
-            $ref: "#/definitions/ErrorResponse"
-
-  /church/matchgroups: # church and user are determined based on user session
-    get:
-      tags:
-      - "leader"
-      summary: "Retrieve list of all your church match groups"
-      description: "Retrieve all match groups of a specific church."
-      operationId: "getMatchGroups"
-      parameters:
-      - in: "path"
-        name: "churchUuid"
         type: "string"
         format: "uuid"
-        required: true
       responses:
         200:
           description: "Match group list"
           schema:
             type: "array"
             items:
               $ref: "#/definitions/Group"
+        400:
+          $ref: "#/responses/ClientError"
+        401:
+          $ref: "#/responses/Unauthenticated"
         404:
           description: "Church not found"
           schema:
-            $ref:  "#/definitions/ErrorResponse"
-            
+            $ref:  "#/definitions/Error"
+
   /user:
     post:
       tags:
-      - "member"
-      summary: "Create new user account"
-      description: "Create a new user to be authenticated via Firebase. Both church leaders and members need such an account."
-      operationId: "createUser"
+      - "administrator"
+      summary: "Create new user"
+      description: "Add a new user to the system."
+      operationId: "addUser"
       consumes:
       - "application/json"
       produces:
       - "application/json"
       parameters:
       - in: "body"
         name: "body"
-        description: "User details for new account"
+        description: "User details"
         required: true
         schema:
           $ref: "#/definitions/UserIn"
       responses:
         200:
-          description: "User account created"
+          description: "User added"
           schema:
-            $ref: "#/definitions/User"
+            $ref: "#/definitions/Question"
         400:
-          description: "Invalid input"
-          schema:
-            $ref: "#/definitions/ErrorResponse"
-      
-  /user/{userUuid}:
+          $ref: "#/responses/ClientError"
+        401:
+          $ref: "#/responses/Unauthenticated"
+        403:
+          $ref: "#/responses/Unauthorized"
     get:
       tags:
-      - "member"
-      summary: "Retrieve user details"
-      description: "Retrieve user details."
-      operationId: "getUser"
-      parameters:
-      - in: "path"
-        name: "userUuid"
-        type: "string"
-        format: "uuid"
-        required: true
+      - "leader"
+      summary: "Retrieve users"
+      description: "Retrieve all users (administrator) or those associated with church (leader)."
+      operationId: "getUsers"
```

Comment:
Should we separate these into 2 different endpoints? So that one is for leaders to get questions for their specific church for a given UUID? To remain consistent with our paradigm

Reply:
Yes, we can do that. However, if you hit `/users` and aren't an administrator, do you just get a 403?

Reply:
Yes that would make sense